packages/composables/src/useCartItem.ts

```diff
@@ -37,6 +37,10 @@ export type UseCartItemReturn = {
    * Determines if the current item is a promotion
    */
   isPromotion: ComputedRef<boolean>;
+  /**
+   * Determines if the current item can be removed from cart
+   */
+  isRemovable: ComputedRef<boolean | undefined>;
```

Comment:
we can do this as boolean as it's used in v-if either way
```suggestion
  isRemovable: ComputedRef<boolean>;
```